package.json

```diff
@@ -4,18 +4,23 @@
   "author": "Ionic Framework",
   "homepage": "https://ionicframework.com/",
   "scripts": {
-    "ng": "ng",
-    "start": "ng serve",
-    "build": "ng build",
     "test": "ng test --no-watch",
     "test:watch": "ng test",
     "test:coverage": "ng test --no-watch --code-coverage",
+    "remote-devtools-server": "./bin/remote-devtools-server.sh",
+    "serve:local": "npm run config:local && ionic serve",
+    "serve:emulator": "npm run config:dev-tools && ionic cap run ios",
+    "serve:ipad": "npm run config:dev && npm run schema-version",
```

Comment:
If this `serve:ipad` script doesn't do what it says it does that please replace it with a text indicating that it's not complete